workflows/parameterizing-a-workflow.md

```diff
@@ -0,0 +1,16 @@
+# Workflow Parameters
+
+Oftentimes, arguments to workflows might not be known until runtime. A workflow might even need to be run multiple times with different parameters.
+
+We allow you to parameterize your workflows with parameter artifacts, which are just a type of artifact that wraps a mutable, jsonable value.
+They are built into your workflow just like any other artifact. However, their values can be adjusted and workflows re-run with the new values.
+
+See this [**Workflow Tutorial**](example-workflows/using-workflow-parameters.md) for a walkthrough of the API.
```

Comment:
I think we need a little bit more description here. We don't need super comprehensive code snippets like you have in the notebook, but we should cover the basics here. Here's some things I can imagine users want to know:

* Are there type constraints on parameters? Is it only primitives, or do we support arbitrary objects?
* What's the basic syntax for defining a parameter? I know this is covered in the notebook, but you can imagine someone not wanting to click through to another page in order to try it out.
* Are parameters always required to have default values? 
* How are parameters passed into functions? SQL queries? Since parameters are artifacts, do I have to pass them into the artifacts list in `publish`?
* How do parameters interact with previews?